### PR TITLE
updated_to_new_logo

### PR DESCRIPTION
**Pull request by indra057**
can u see if i need to change any other logo ..u can reply here..also tell me if i am wrong anywhere i shall correct those...and if u merge it plzz merge after 30 september so that i can come under hacktoberfest(if u want to ,,if u are busy u can merge now)

### CODE REVIEW

File: src/main/resources/icons/icon80x80.png
Comment (andrii-bodnar) [inline src/main/resources/icons/icon80x80.png:1]: This icon should be 80x80 in PNG format. But if to see the content of the image in your branch - it's CSV inside. It won't work this way. Please use some online converters from SVG to PNG.

File: src/main/resources/icons/icon_dark.svg
Comment (andrii-bodnar) [inline src/main/resources/icons/icon_dark.svg:1]: The image title is a little misleading. Please use the **light** "_Mark_" icon from the Media kit here.

Comment (indra057) [reply src/main/resources/icons/icon_dark.svg:1]: sir i didn't understood what u are telling ...should i change the icon_dark.svg  whole file ??can u plzz describe a little bit as this is my first contribution and i am little confused..

Comment (andrii-bodnar) [reply src/main/resources/icons/icon_dark.svg:1]: Just update it with the same icon as you did but use white variant, not dark

File: src/main/resources/icons/icon80x80.png
Comment (indra057) [reply src/main/resources/icons/icon80x80.png:1]: i changed it..

File: src/main/resources/icons/icon_dark.svg
Comment (indra057) [reply src/main/resources/icons/icon_dark.svg:1]: sir if i use the white variant icon.svg and icon_dark.svg will be same i think ..is it ok to change it??

Comment (andrii-bodnar) [reply src/main/resources/icons/icon_dark.svg:1]: Keep the current image name as is

Comment (indra057) [reply src/main/resources/icons/icon_dark.svg:1]: i changed it. could u please review it

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**